lib/Nexus.js

```diff
@@ -268,6 +269,42 @@ module.exports = function xGraph(__options = {}) {
 				return pid;
 			}
 
+			/**
+			 * Scan the array of Pids and send a 'GetServices'
+			 * command to each in turn to accumulate the 
+			 * services used by a partifcular module
+			 * @param {array} services		array of Pids to send GetServices to
+			 */
+			async function getServices(services) {
```

Comment:
this parameterization should probably be written using a spread operator, so it may be called with the more standard: 
```
getServices(pid)
// or
getServices(pid1, pid2, pid3)
```

as opposed to requiring the end user to include the array brackets themselves.